sdk/openai/openai/README.md

```diff
@@ -250,6 +252,52 @@ async function main(){
 }
 ```
 
+### Analyze Business Data
+
+This example generates chat responses to input chat questions about your business data. The business data is provided through an Azure Cognitive Search index. To learn more about how to setup an Azure Cognitive Search index as a data source, see [Quickstart: Chat with Azure OpenAI models using your own data][msdocs_quickstart_byod].
+
+
+```javascript
+const { OpenAIClient } = require("@azure/openai");
+
+async function main(){
+  const endpoint = "https://myaccount.openai.azure.com/";
+  const client = new OpenAIClient(endpoint, new DefaultAzureCredential());
+
+  const deploymentId = "gpt-35-turbo";
+
+  const messages = [
+    { role: "user", content: "What's the most common customer feedback about our product?" },
+  ];
+
+  console.log(`Messages: ${messages.map((m) => m.content).join("\n")}`);
+
+  const events = client.listChatCompletions(deploymentId, messages, { 
```

Comment:
the response looks like it is streamed, but we don't pass the `stream: True` parameter? Is this just how JS works?

Reply:
There are two chat completions methods, `getChatCompletions` that returns a single response object, and `listChatCompletions` that returns a stream of completion events. The latter sets the stream option internally.